<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClipGrab - Download and Convert Online Videos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget
{
background:#fbfbfc
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="4" column="1">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:none;</string>
      </property>
      <property name="text">
       <string>Â© 2009 Philipp Schmieder | http://clipgrab.de</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTabWidget" name="mainTab">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style: solid;
border-width: 3px;
border-radius: 4px;
color: white;
font: bold 14px;
border-radius: 10px;
padding-top:4px;
padding-bottom:4px;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.386, stop:0 rgba(0, 178, 222, 255), stop:1 rgba(65, 191, 222, 255));
border-color:rgb(0, 178, 222);
}
QPushButton:hover{
background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.386, stop:0 rgba(53, 215, 255, 255), stop:1 rgba(65, 191, 222, 255))
}
QPushButton:pressed{
background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.386, stop:0 rgba(53, 215, 255, 255), stop:1 rgba(65, 191, 222, 255));
border-color: rgb(12, 206,255);
}</string>
       </property>
       <attribute name="title">
        <string>Start</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="1">
         <widget class="QPushButton" name="gotoSearch">
          <property name="text">
           <string>Search for videos</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="gotoDownload">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Start a new Download</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="startIntoduction">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Welcome to the latest version of ClipGrab!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;With ClipGrab you can download video files from almost any online video platform&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Downloads</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="5" column="0" colspan="5">
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">#scrollAreaWidgetContents
{
background:transparent;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>626</width>
             <height>187</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0" colspan="3">
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>Current Downloads:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QTreeWidget" name="downloadTree">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="columnCount">
               <number>4</number>
              </property>
              <column>
               <property name="text">
                <string>Portal</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Title</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Format</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Progress</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="downloadStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Grab this clip!</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="downloadComboQuality">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="downloadComboFormat">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QLineEdit" name="downloadLineEdit"/>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QLabel" name="downloadInfoBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#00b2de;
color:#ffffff;
padding:4px;</string>
          </property>
          <property name="text">
           <string>Please enter the link to the video you want to download in the field below.</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>Enter your seach and hit enter to start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="searchPortalCombo">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Target path</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0" colspan="4">
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>48</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>48</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#00b2de;
color:#ffffff;
padding:4px;</string>
              </property>
              <property name="text">
               <string>Here you can configure where the downloaded videos are to be saved.</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="settingsSavedPath">
              <property name="toolTip">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="settingsBrowseTargetPath">
              <property name="text">
               <string>Browse ...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Target path</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QCheckBox" name="settingsSaveLastPath">
              <property name="text">
               <string>Always save the last used path.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Conversion</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="4" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>191</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>48</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#00b2de;
color:#ffffff;
padding:4px;</string>
              </property>
              <property name="text">
               <string>Experts can edit or create new presets for the video conversion here.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Format preset</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Add new preset</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Remove selected preset</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="4">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Audio settings</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox_2"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Codec</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Bitrate (kb/s)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QSpinBox" name="spinBox"/>
               </item>
               <item row="0" column="2">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="6">
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>Disable Audio</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Video settings</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="QComboBox" name="comboBox_3"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dadadf, stop:1 #fbfbfc);
padding-top:10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/img/logo.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
